Author: Agustin Martin Domingo <agmartin@debian.org>
Description: Make sure all filters stuff is installed together, allowing separate datadir and libdir.
Bug-Debian: http://bugs.debian.org/612051
Applied-Upstream: Reported

--- a/Makefile.am
+++ b/Makefile.am
@@ -272,8 +272,8 @@ ddtp_filter_la_LDFLAGS  = ${filter_ldfla
 endif # COMPILE_IN_FILTERS
 
 noinst_DATA += $(static_optfiles) gen/filter.pot
-opt_DATA = $(dynamic_optfiles)
-filter_DATA = $(fltfiles)
+# opt_DATA = $(dynamic_optfiles)
+filter_DATA = $(fltfiles) $(dynamic_optfiles)
 
 # settings.h added as a dependency so it will get recreated if
 #   the COMPILE_IN_FILTERS option changes
